src/main/java/com/epam/test_generator/config/security/WebSecurityConfig.java

```diff
@@ -106,6 +106,7 @@ public CorsConfigurationSource corsConfigurationSource() {
      */
     @Override
     public void configure(WebSecurity web) throws Exception {
+        //todo тут некоторое дублирование с методом configure(HttpSecurity http)
```

Comment:
these todos may cause merge conflicts with other websecurity related branch. Better roll it back.